code/game/objects/items/hand_item.dm

```diff
@@ -0,0 +1,87 @@
+/obj/item/slapper
+	name = "slapper"
+	desc = "This is how real men fight."
+	icon_state = "latexballon"
+	item_state = ""
```

Comment:
this has to be this way; if it's `null`, it'll inherit the `icon_state` as the default `item_state`